komodo/lint_package_status.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+
+import yaml
+
+
+VALID_VISIBILITY = ["public", "private"]
+VALID_IMPORTANCE = ["low", "medium", "high"]
+VALID_MATURITY = ["experimental", "stable", "deprecated"]
+
+
+def is_valid_option(status, field, valid_options):
+    value = status.get(field)
+    return value is not None and value in valid_options
+
+
+def run(package_status, repository):
+    package_status_set = set(package_status.keys())
+    repository_set = set(repository.keys())
+    if package_status_set.difference(repository_set):
+        raise SystemExit(
+            "The following packages are specified in the package status file, but not in the repository file: {}".format(
+                list(package_status_set.difference(repository_set))
+            )
+        )
+    if repository_set.difference(package_status_set):
+        raise SystemExit(
+            "The following packages are specified in the repository file, but not in the package status file: {}".format(
+                list(repository_set.difference(package_status_set))
+            )
+        )
+
+    errors = []
+    for package, status in package_status.items():
+        if not is_valid_option(status, "visibility", VALID_VISIBILITY):
```

Comment:
could be simplified to 
```suggestion
      if not status.get('visibility') in VALID_VISIBILITY
```